spark-kafka-0-10-writer/src/main/scala/com/github/benfradet/spark/kafka010/writer/DStreamKafkaWriter.scala

```diff
@@ -41,10 +41,11 @@ class DStreamKafkaWriter[T: ClassTag](@transient private val dStream: DStream[T]
    */
   override def writeToKafka[K, V](
     producerConfig: Properties,
-    transformFunc: T => ProducerRecord[K, V]
+    transformFunc: T => ProducerRecord[K, V],
+    callback: Option[Callback]
```

Comment:
Do you think this could have `None` as a default value?

This would avoid breaking source code compatibility.